src/create.rs

```diff
@@ -125,22 +161,29 @@ fn run_container<P: AsRef<Path>>(
             let without = sched::CloneFlags::CLONE_NEWUSER;
             namespaces.apply_unshare(without)?;
 
+            // set up tty if specified
             if let Some(csocketfd) = csocketfd {
                 tty::ready(csocketfd)?;
             }
 
+            // set namespaces
             namespaces.apply_setns()?;
 
+            // fork second time, which will later create container
             match fork::fork_init(child)? {
-                Process::Child(child) => Ok(Process::Child(child)),
+                Process::Child(child) => unreachable!(),
```

Comment:
The variable `child` is not used in this line. Therefore, the `child` variable needs to be renamed to `_child`.